### PR TITLE
Fix the README.md to explain the Known client applications

### PR DESCRIPTION
**Pull request by jmprieur**
- Now that known client applications is available for MSA, adding it, by updating
   the README.md
- Updating the service to send the URI for MSA users to consent
  to the service calling Microsoft Graph (by sending a wwwAuthenticate
  message back to the client, which handles the consent Url
- Adding an index.html static file to the service (when the consent is done)
- Updating the client to handle the wwwAuthenticate header
  by displaying a dialog box to the user, and then by launching
  a browser to let the user consent.
- fixing a couple of issues in the client.

## Purpose
Explain better the experience with Web APIs in multi-tenants, and with MSA
* ...

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ x] Documentation content changes
[ ] Other... Please describe:
[ x] evolution with the service changes

```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]

```

* Test the code
Follow the updated README.md
```
```

## What to Check
- The developer can use a tenanted authority and a scope of api://{clientId-of-service}/**.default**, and sign-in users from the tenant and the client calls the web API
- The developer can use a TenantId of 'organizations' and a scope of api://{clientId-of-service}/.default, and sign-in users from any tenant and the client calls the web API
- The developer can use a TenantId of 'common' and a scope of api://{clientId-of-service}/**access_as_user**, and sign-in users from any tenant or Personal account, and the client calls the web API. However user of personal accounts will need to consent to the service accessing their provfile (there is a sign-up to the service step)

* ...

### CODE REVIEW

**Comment by jmprieur** [comment]
FYI, @jennyf19, I'll incorporate the feedback from your PR after this one and after merging Microsoft.Identity.Web from the ASP.NET Core Web app tutorial

File: 2. Web API now calls Microsoft Graph/README.md [hunk 1]
```diff
@@ -256,8 +260,10 @@ Note: if you used the setup scripts, the changes below will have been applied fo
 1. In the TodoListClient project, open `App.config`.
 1. Find the app key `ida:ClientId` and replace the value with the ApplicationID (Client ID) for the *TodoListClient-v2* app copied from the app registration page.
 1. Find the app key `todo:TodoListScope` and replace the value with the scope of the TodoListService-v2 application copied from the app registration in the **Expose an API** tab, but replace the scope by `.default` (of the form ``api://<Application ID of service>/.default`` if you followed the instructions above)
+   > Important: If you want don't want to sign-in Microsoft personal accounts, use the `api://<Application ID of service>/.default` and not `api://<Application ID of service>/.access_as_user`, so that the permissions of the Web API get rolled-up in the client's consent screen. If you do want to sign-in users with personal accounts (TenantId=`common`), then keep `api://<Application ID of service>/.access_as_user`, but users who sign-in from other tenants or with personal account will need to consent to the Web API accessing their profile
```
Comment (pobisw) [inline 2. Web API now calls Microsoft Graph/README.md]: If you don't want

Comment (jmprieur) [reply 2. Web API now calls Microsoft Graph/README.md]: Thanks. Adressed.

**Comment by kalyankrishna1** [comment]
@kalyankrishna1 is added to the review. #Closed

**Comment by jmprieur** [comment]
Thanks @TiagoBrenck 
- for the first issue, do you have cycles to propose a PR to fix the script?
- the second is expected for the moment (KCA does not work yet with MSA)

The 3rd chapter is really client and service have the same ClientID. Maybe we shoud name it that way? KCA for MSA will be released at some point, and things will be easier to explain.

**Comment by TiagoBrenck** [comment]
@jmprieur I will open a PR to fix the scripts. However I am still not able to successfully run an E2E test  using a member user. Could you double check that this sample is working for you?

**Pull request merged**